src/main/java/com/minecolonies/coremod/tileentities/TileEntityColonyBuilding.java

```diff
@@ -23,6 +23,7 @@
      * NBTTag to store the colony id.
      */
     private static final String TAG_COLONY = "colony";
+    private static final String TAG_MIRROR = "mirror";
```

Comment:
Because we can get the rotation from the block because of its facing. But we can't get the mirror from it.